packages/dd-trace/src/appsec/iast/utils.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+function iterateObjectStrings (target, fn, levelKeys = [], depth = 20, visited = new Set()) {
+  if (target !== null && typeof target === 'object') {
+    Object.keys(target).forEach((key) => {
```

Comment:
NIT: Depth is protecting us from a circular objects reference, It would be safer if we move `visited.add(val) `before recursion to prevent possible infinite loops

```suggestion
    if (visited.has(target)) return
    visited.add(target)
    
    Object.keys(target).forEach((key) => {
```